logdevice/ops/ldquery/tables/LogTreeBase.cpp

```diff
@@ -0,0 +1,81 @@
+#include "LogTreeBase.h"
+#include <folly/Conv.h>
+#include <folly/json.h>
+#include "../Utils.h"
+#include "logdevice/common/configuration/ReplicationProperty.h"
+
+namespace facebook::logdevice::ldquery::tables {
+
+TableColumns LogTreeBase::getAttributeColumns() {
+  return {
+      {"name", DataType::TEXT, "Name of the log group or the directory."},
+      {"logid_lo", DataType::LOGID,
+       "Defines the lower bound (inclusive) of the range of log ids in this "
+       "log group."},
+      {"logid_hi",
+       DataType::LOGID,
+       "Defines the upper bound (inclusive) of the range of log ids in this "
+       "log group."},
+      {"replication_property",
+       DataType::TEXT,
+       "Replication property configured for this log group."},
+      {"synced_copies",
+       DataType::INTEGER,
+       "Number of copies that must be acknowledged by storage nodes are synced "
+       "to disk before the record is acknowledged to the client as fully "
+       "appended."},
+      {"max_writes_in_flight",
+       DataType::INTEGER,
+       "The largest number of records not released for delivery that the "
+       "sequencer allows to be outstanding."},
+      {"backlog_duration_sec",
+       DataType::INTEGER,
+       "Time-based retention of records of logs in that log group.  If null or "
+       "zero, this log group does not use time-based retention."},
+      {"storage_set_size",
+       DataType::INTEGER,
+       "Size of the storage set for logs in that log group.  The storage set "
+       "is the set of shards that may hold data for a log."},
+      {"delivery_latency",
+       DataType::INTEGER,
+       "For logs in that log group, maximum amount of time that we can delay "
+       "delivery of newly written records.  This option increases delivery "
+       "latency but improves server and client performance."},
+      {"scd_enabled",
+       DataType::INTEGER,
+       "Indicates whether the Single Copy Delivery optimization is enabled for "
+       "this log group.  This efficiency optimization allows only one copy of "
+       "each record to be served to readers."},
+      {"parent_dir", DataType::TEXT, "Name of the parent directory."},
```

Comment:
Not sure we actually need `parent_dir` as it contained in name (it is in fact fully-qualified path).